src/main/java/com/coherensolutions/training/automation/java/web/urnezaite/LogInPage.java

```diff
@@ -0,0 +1,51 @@
+package com.coherensolutions.training.automation.java.web.urnezaite;
+
+import com.coherensolutions.training.automation.java.web.urnezaite.util.DriverManager;
+import com.coherensolutions.training.automation.java.web.urnezaite.util.PropertyProvider;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+public class LogInPage extends DriverManager {
+    private WebDriver driver;
+    private final String USERNAME = PropertyProvider.getProperty("username");
+    private final String PASSWORD = PropertyProvider.getProperty("password");
+    private final String LOGIN_PAGE_LINK = "http://automationpractice.com/index.php?controller=authentication&back=my-account";
```

Comment:
let's have those variables in the test class, make your page methods flexible as much as possible